backend/python/app/resources/entity_dto.py

```diff
@@ -23,3 +23,22 @@ def __init__(
         self.string_array_field = string_array_field
         self.enum_field = enum_field.upper() if enum_field is not None else enum_field
         self.bool_field = bool_field
+
```

Comment:
Could you modify the `EntityDTO` constructor above to use `kwargs` as well, and remove the previous validations that threw Exceptions (I forgot that they were there)